### PR TITLE
[SINGULAR] PS: Cannot load contact information form after AOM

### PR DESCRIPTION
**Pull request by juanseba21**
'Provide Contact Information' form is now displaying properly:

Before the fix:

image

After the fix:

image

### CODE REVIEW

File: ddp-workspace/projects/ddp-singular/src/app/constants/activity-code.ts [hunk 1]
```diff
@@ -12,4 +12,5 @@ export enum ActivityCode {
   AddParticipantParental = 'ADD_PARTICIPANT_PARENTAL',
   AddParticipantDependent = 'ADD_PARTICIPANT_DEPENDENT',
   MedicalRecordFileUpload = 'MEDICAL_RECORD_FILE_UPLOAD',
+  ChildContact = 'CHILD_CONTACT'
```
Comment (juanseba21) [inline ddp-workspace/projects/ddp-singular/src/app/constants/activity-code.ts:15]: Adding CHILD_CONTACT as an activity-code

File: ddp-workspace/projects/ddp-singular/src/app/utils/helpers.ts [hunk 2]
```diff
@@ -35,6 +35,8 @@ const getRenderActivityKeyByActivityCode = (activityCode: ActivityCode): RenderA
       return RenderActivityKey.MedicalRecordUpload;
     case ActivityCode.PatientSurvey:
       return RenderActivityKey.PatientSurvey;
+    case ActivityCode.ChildContact:
+      return RenderActivityKey.About;
```
Comment (juanseba21) [inline ddp-workspace/projects/ddp-singular/src/app/utils/helpers.ts]: Returning About as a Render-Activity-Code

Comment (alextanlan) [reply ddp-workspace/projects/ddp-singular/src/app/utils/helpers.ts]: I believe you have caught the bug root cause.

**One note**:  suggest moving that check to line 27 where the condition has already been returning `RenderActivityKey.About`

**Comment by MocanaAtBroad** [comment]
This seems like it fixes problem. Only thing I am not 100% sure is if it is correct what the wizard looks like, i.e., the activity asking for the child's email appears in wizard under "About My Child".
I am inclined to say, let's merge to get fix in and then let's verify fix with Whitney and the rest.

**Review by MocanaAtBroad — DISMISSED** [review]

File: ddp-workspace/projects/ddp-singular/src/app/utils/helpers.ts (hunk 2 above)
Comment (juanseba21) [reply ddp-workspace/projects/ddp-singular/src/app/utils/helpers.ts]: sure, Makes sense.

**Review by alextanlan — APPROVED** [review]

**Pull request merged**